TrainingRNExpress/src/app/screens/TodoList/layout.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Button, Text, TouchableOpacity, TextInput, FlatList } from 'react-native';
+
+import { TEXT_PLACEHOLDER, BUTTON_LABEL } from './strings.js';
+import { RED } from '../../../constants';
+import Item from '../../components/Item';
+import styles from './styles.js';
+
+const TodoList = ({
+  items,
+  newItem,
+  onChangeText,
+  onCreateItem,
+  onItemPress,
+  onItemRemove,
+  cleanItems
+}) => {
+  const renderItem = ({ item, index }) =>
+    <Item
+      item={item}
+      onPress={() => onItemPress(index)}
+      onRemove={() => onItemRemove(index)}
+    />
+
+  const keyExtractor = (_, index) => index.toString();
```

Comment:
I'd convert this component as a Class so we can extract these functions to instance methods and create them only once. What do you think ?